### DISCUSSION TITLE
Is version 7.x.x Supported? Particularly the MudExThemeEdit?

### DISCUSSION THREAD

**Discussion by aarondglover** [fgilde/MudBlazor.Extensions#84] [Q&A] [answered]
Hi All

Do any of the extension in this library work with Mudblazor v7.x.x?

I am particularly interested in the MudExThemeEdit component. I note some breaking changes from v6 to v7 - most noteably the change of name from Pallet to "LightPallet" in order align with the naming convention for 'DarkPallet' etc.

Thanks in Advance

**Comment by fgilde** [comment]
Not yet but I will create a version for mudblazor 7 soon

**Comment by fgilde** [comment, marked as answer]
I started yesterday evening with an update to mudblazor 7. Still not finished and some bugs but a testable pre version is available here https://www.nuget.org/packages/MudBlazor.Extensions/2.0.0-prev-2406301522-for-mudblazor-7.0.0

**Comment by fgilde** [comment]
Small update the current preview version has fixed mostly all important bugs. But many styles are still broken

https://www.nuget.org/packages/MudBlazor.Extensions/2.0.0-prev-2407011629-for-mudblazor-7.0.0

**Comment by versile2** [comment]
Just as an FYI I created a [Theme Creator](https://themes.arctechonline.tech/) for MudBlazor that feels amazing to use. I've also started with all the themes from Bootstrap for easy use. I just added the MudExDialog because draggable Dialogs??? I think so. And found on another thread why it was flickering on first open.
